### DISCUSSION TITLE
G6 Sensor stops quietly

### DISCUSSION THREAD

**Discussion by andyrozman** [NightscoutFoundation/xDrip#3000] [Q&A] [answered]
Hi !
So far last few months, my sensors stops without notifying me. 

Is there alarm I can set that will let me know that sensor has stopped? I have been using Xdrip for a long time now, and maybe I have disabled this alarm in the past (for whateever reason). 

Or even better question. Is there alarm/notification I can set that will give me warning even before that? So that I can get prepared?  So that I would get notification, something like "Your sensor is about to expire in 24/12/6 hours". 

Kind regards,
Andy

**Comment by jonprogers** [comment]
Although I don't use it, there is a 'Sensor Expiry' alert in Settings -> Alarms & Alerts -> Extra Alerts. The description is 'Raise notifications when the sensor gets close to expiry' which sounds close to what you are looking for...

**Comment by Navid200** [comment, marked as answer]
1- There is a missed readings alert.  It will alert you if your readings stop no matter for what reason, sensor stopping or any other reason.
2- There is a sensor expiry alert.  It does not make a sound though.  It will notify you 24 hours, 9 hours and 2 hours before expiry.

 https://navid200.github.io/xDrip/docs/Alerts_page.html

**Comment by andyrozman** [comment]
Thank you, that was exactly what I was looking for.
